client/components/EventForm/fieldRenders.jsx

```diff
@@ -3,88 +3,97 @@ import React from 'react'
 import { fields } from '../index'
 import { Field, FieldArray } from 'redux-form'
 
-const renderSlugline = (readOnly) => (
+
+const renderSlugline = (args) => (
```

Comment:
maybe it could be called `props` instead of `args` to be more consistent with other components